start-from-zero/Javascript/src/views/evaluation/evaluation.js

```diff
@@ -0,0 +1,122 @@
+function EvaluationsPage(options){
+    const headings = ['Name','Language','Level',''];
+    
+    const rows = [
+        {
+            name:'Dan Constantinescu',
+            language: 'Javascript',
+            level:'Junior'
+        },
+        {
+            name:'Adolf Hgg',
+            language: 'SwS',
+            level:'Master'
+        },
+        {
+            name:'Fiodor D',
+            language: 'Russian',
+            level:'Master'
+        },
+    ];
+
+    const newOptions = options || {};
+    return `
+    ${NAV()}
+    ${EvaluationsTable({
+        items: rows,
+        itemHeadings: headings
+    })}
+    ${Footer()}
+
+
+    `;
+}
+
+function EvaluationTableHeader(options){
+    const headings = [];
+    options.headings.forEach(function(el){
+        const generatedEl = `<th class='table-head-element'>${el}</th>`;
+        headings.push(generatedEl);
+        
+    });
+
+    const headingsEl = `${headings.join('')}`;
+
+    return `
+    <thead>
+        <tr class="table-head-row">
+            ${headingsEl}
+        </tr>
+    </thead>
+    `;
+}
+
+
+
+
+function EvaluationTableRow(options={}){
+
+    return `
+    <tr class='table-body-tr'>
+        <td>${options.name}</td>
+        <td>${options.language}</td>
+        <td>${options.level} 2</td>
+        <td class='detalii'>Detalii 
+            <a href="#" class="button-toggle plus-button">
+                <span class="minus-toggle"></span>
```

Comment:
Hmmm... so there's a `.minus-toggle` inside of a `.plus-button`... am I the only one to whom this looks off?